packages/v4/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,345 @@
+                <Text className="pf-v5-u-content-bold">
+                  {" "}
+                  Design and build better product experiences in the
+                  <Text component="span" className="pf-v5-u-primary-color-100">
+                    {" "}
+                    open{" "}
+                  </Text>
+                  with PatternFly
+                </Text>
+              </Title>
+              <Text component={TextVariants.p} className="ws-mdx-p">
+                PatternFly is an open source design system that enables teams to
+                create consistent and scalable enterprise products. PatternFly
+                is sponsored and maintained by Red Hat, but is open to all.
+              </Text>
+            </Flex>
+          </TextContent>
+          <Flex
+            flexWrap={{default: "nowrap"}}
+            spaceItems={{default: "spaceItemsMd"}}
+          >
+            <FlexItem>
+              <Link
+                to="/get-started/design"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start designing"
+              >
+                Start designing
+              </Link>
+            </FlexItem>
+            <FlexItem>
               <Link
-                to="/get-started/about"
-                className="pf-v5-c-button pf4-c-button__cta-primary pf-v5-u-mr-0 pf-v5-u-mb-md pf-v5-u-mb-0-on-md pf-v5-u-mr-xl-on-md fadeIn animated fadeInFour"
-                aria-label="Get started with PatternFly"
+                to="/get-started/develop"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start developing"
               >
-                Get started
+                Start developing
               </Link>
-              <a className="pf-v5-c-button pf4-c-button__cta-outline fadeIn animated fadeInFour" href="//github.com/patternfly" target="_blank">
-                <GithubIcon className="pf-v5-u-mr-sm" />GitHub
+            </FlexItem>
+          </Flex>
+        </Flex>
+        <Flex flex={{sm: "flexNone", md: "flex_3", lg: "flex_3"}}>
+          <img
+            className="pf-v5-c-image__light-showcase"
+            alt="Light theme image showcase"
+            src={lightshowcase}
+          />
+        </Flex>
+      </Flex>
+    </PageSection>
+    <Divider inset={{ default: "inset2xl"}} />
+    <PageSection className="v5-featured-posts">
+      <Flex flexWrap={{default: "wrap"}}>
+        <FlexItem flex={{default: "flex_1"}}>
+          <Title headingLevel="h1">Featured blog posts</Title>
+        </FlexItem>
+        <Button
+          isInline
+          size="lg"
+          component="a"
+          href="https://medium.com/patternfly"
+          variant="link"
+          aria-label="explore our blog"
+          style={{float: "right"}}
+        >
+          Explore our blog <ArrowRightIcon />
+        </Button>
+      </Flex>
+    </PageSection>
+    <PageSection>
+      <Gallery
```

Comment:
Looks like we can use `Grid` here instead

```suggestion
      <Grid hasGutter md={6} xl={4}>
```